lib/YuiRestClient.pm

```diff
@@ -13,12 +13,18 @@ package YuiRestClient;
 use strict;
 use warnings;
 
+use constant API_VERSION => 'v1';
+
+use testapi;
+use utils 'type_string_slow';
+use Utils::Backends 'is_pvm';
+use YuiRestClient::App;
+
+
 our $interval = 1;
 our $timeout  = 10;
```

Comment:
these two variables are not needed anymore once App contains all the info, right? or we can pass it in the constructor I guess.

Reply:
Yeah, problem is that once I tried to reuse `lib/YuiRestClient.pm` it created circular dependencies, as originally we would import those values, instead of passing them. I would keep it as it is, as we still use them and they can be set here to be used everywhere else. Once we move it to separate perl module, issue would be solved by itself.